integration_tests/data/cross_db/data_listagg.csv

```diff
@@ -0,0 +1,5 @@
+group_col,string_text,order_col
+1,a,1
+1,b,2
+2,c,1
+2,d,2
```

Comment:
I like it when the tests can't just work by accident, e.g. if it happens to pull the data out in the default order, then we don't know whether our ordering is working or not. 

```suggestion
2,d,2
2,c,1
```